pkg/strategy/ewo_dgtrd/strategy.go

```diff
@@ -0,0 +1,409 @@
+package ewo_dgtrd
+
+import (
+	"context"
+	"sync"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/c9s/bbgo/pkg/bbgo"
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/indicator"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+const ID = "ewo_dgtrd"
+
+var log = logrus.WithField("strategy", ID)
+var modifier = fixedpoint.NewFromFloat(0.99)
+
+func init() {
+	bbgo.RegisterStrategy(ID, &Strategy{})
+}
+
+type Strategy struct {
+	*bbgo.Graceful
+	//bbgo.SmartStops
+	market         types.Market
+	session        *bbgo.ExchangeSession
```

Comment:
you can use:

```
	Market         types.Market
	Session        *bbgo.ExchangeSession
```

and these fields will be injected automatically.

Reply:
done